AppServiceSharedNotepad/CS/README.md

```diff
@@ -0,0 +1,322 @@
+# Using an app service with multiple clients
+We’ll create a simple notepad [app service](https://docs.microsoft.com/en-us/windows/uwp/launch-resume/how-to-create-and-consume-an-app-service).  We'll also create two simple app service clients. One that writes to the notepad service and one that reads from the notepad service.
+
+## Load the projects in Visual Studio
+___
+
+* You can find the source code for this sample by downloading a zip of all of our samples in WIndows 10 IoT Enterprise folder and navigating to the `samples-develop\AppServiceBlinky`. 
+* Make a copy of the folder on your disk and open the projects from Visual Studio. 
+* NotepadService.sln implements the app service and must be started first.
+* NotepadClientBackgroundApplication.sln implements a client that writes messages to the app service.
+* NotepadServiceClientApp.sln implements a windows that retrieves messages from the client and displays them.
+* Make a copy of the folder on your disk and open the projects from Visual Studio. 
+* NotepadService.sln implements the app service and must be started first. 
+* NotepadClientBackgroundApplication.sln implements the app service client.
+
+## Deploy your app
+
+* * *
+
+With the application open in Visual Studio 2019, set the architecture in the toolbar dropdown. If you’re building for UpBaord, select `x64`.

```

Comment:
Misspelled "UpBaord", please use "UP Board" across all files.

Reply:
Updated it